### PR TITLE
[MOB-15818] Added objective-c samples for public classes

### PR DESCRIPTION
**Pull request by addb**
Addition to https://github.com/adobe/aepsdk-edgeidentity-ios/pull/65 PR

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Documentation/api-reference.md [hunk 1]
```diff
@@ -385,8 +385,9 @@ For more information, please read an overview of the [AEP Identity Service](http
   }
 ```
 
-**Example**
+##### Example
```
Comment (cacheung) [inline Documentation/api-reference.md:388]: Can we keep the same format
Swift
Example

Objective-C
Example

File: Documentation/api-reference.md [hunk 2]
```diff
@@ -431,6 +456,18 @@ let state: AuthenticatedState = item.authenticatedState
...
+// Initialize
+AEPIdentityItem* item = [[AEPIdentityItem alloc] initWithId:@"identity" authenticatedState:AEPAuthenticatedStateAuthenticated primary:false];
+
+// Getters
+NSString* id = primaryEmail.id;
+
+long state = primaryEmail.authenticatedState;
+
+bool primary = primaryEmail.primary;
+```
```
Comment (cacheung) [inline Documentation/api-reference.md:470]: The "**Syntax**" in line 483 needs be be reformatted.

**Review by cacheung — COMMENTED** [review]
For public classes section, can we keep the same format
####

File: Documentation/api-reference.md [hunk 3]
```diff
@@ -385,8 +385,9 @@ For more information, please read an overview of the [AEP Identity Service](http
   }
```
Comment (emdobrin) [inline Documentation/api-reference.md:385]: One quick comment for the prerequisites section, can you rephrase `Refer our Getting Started Guide<link>` to `Refer to the Getting Started Guide<link>.`

**Review by emdobrin — APPROVED** [review]
Just a small comment, looks good.

**Review by cacheung — APPROVED** [review]
Looks good other than one more format change.

File: Documentation/api-reference.md (hunk 3 above)
Comment (emdobrin) [reply Documentation/api-reference.md:385]: Refer to the ..
